_includes/configuration/advtable_value_series.md

```diff
@@ -0,0 +1,89 @@
+## `advtable_value_series`
+{{ site.requires_5_9v }}
+
+{% if pluginname != "Advanced Tables" %}
+> **Note**: The `advtable_value_series` option requires the Advanced Tables plugin.
+{% endif %}
+
+The `advtable_value_series` option configures value series for populating cells in a table. This option can be used to create row identifiers.
+
+**Type:** `Object`
+
+**Default Value:**
+
+```js
+{
+  // Natural number series
+  numeric: { 
+    title: 'Numeric',
+    update: true,
+    resizable: false,
+    generator: `GeneratorFunction` // For details, see: 'Creating a value series generator'
+  },
+  // English alphabetic series
+  alpha: {
+    title: 'Alpha',
+    update: true,
+    resizable: false,
+    generator: `GeneratorFunction` // For details, see: 'Creating a value series generator'
+  },
+}
+```
+
+Both default series are configured to update on table changes and not resize when using the resize bars.
+
+### Series configuration
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| title | `string` | Required | The name shown in the UI for the series. |
+| update | `boolean` | Optional | default: `false` - When `true`, the series values will be updated when changes are made to the table. |
+| resizable | `boolean` | Optional | default: `true` - When `true`, table cells containing the series values can be resized using a mouse or touch device. |
```

Comment:
Wondering whether we need to specify the input types

```suggestion
| resizable | `boolean` | Optional | default: `true` - When `true`, table cells containing the series values can be resized. |
```

Reply:
I think the input types was something Tyler had put in one of the suggestions in the original PR. I don't have any issues with keeping or removing it personally.

Reply:
My preference would be to remove it as well, but I don't have any strong opinions about it.

Reply:
let's leave them in for clarity.